src/main/resources/templates/change.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Getting Started: Serving Web Content</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <script src="js/app.js" charset="utf-8"></script>
+</head>
+<body>
+<p>
+    Change??
```

Comment:
Ha this was just a template I was using to test, I should get rid of it really. Still always a valid question